AEPCore/Sources/configuration/Configuration.swift

```diff
@@ -44,7 +44,6 @@ class Configuration: NSObject, Extension {
         registerPreprocessor(rulesEngine.process(event:))
 
         registerListener(type: EventType.configuration, source: EventSource.requestContent, listener: receiveConfigurationRequest(event:))
-        registerListener(type: EventType.lifecycle, source: EventSource.responseContent, listener: receiveLifecycleResponse(event:))
```

Comment:
Can you also remove receiveLifecycleResponse function and related code?

Reply:
Oops, thought i did. Thanks, done.